src/test/ui/traits/trait-with-dst-2.rs

```diff
@@ -0,0 +1,20 @@
+struct VTable<DST: ?Sized> {
```

Comment:
I cannot get the ICE to reproduce with godbolt... I tried several versions but no luck.

Reply:
Ah yes, it seems this doesn't cause an ICE (I tested with `nightly-2018-10-23` on my local). Should we remove this test and just close the issue?